### PR TITLE
created movie details page

### PR DESCRIPTION
**Pull request by Deep687**
Created movie-details-page.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2276,6 +2284,23 @@
...
+      "resolved": "https://registry.npmjs.org/axios/-/axios-1.7.7.tgz",
+      "integrity": "sha512-S4kL7XrjgBmvdGut0sN3yJxqYzrDOnivkBiN0OFs6hLiUam3UPvswUo0kqGyhqUZGEOytHyumEdXsAkgCOUf3Q==",
+      "license": "MIT",
+      "dependencies": {
+        "follow-redirects": "^1.15.6",
+        "form-data": "^4.0.0",
+        "proxy-from-env": "^1.1.0"
+      }
+    },
+    "node_modules/axos": {
```
Comment (Vijayshree-ranium) [inline package-lock.json:2298]: axos?

File: src/Views/MovieDetails.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import { useRoute } from "vue-router";
+import { ref, onMounted } from "vue";
+import { fetchMovieImages } from "../utils/fetchMovieImages";
+import { fetchMovieDetails } from "../utils/fetchMovieDetails";
+import { fetchMovieCredits } from "../utils/fetchMovieCredits";
+
+const movieDetails = ref({});
+const movieImages = ref([]);
+const cast = ref([]);
+const castSlice5 = ref([]);
```
Comment (Vijayshree-ranium) [inline src/Views/MovieDetails.vue:62]: Use proper key word

File: src/utils/fetchMovieImages.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  
+      if (!response.ok) {
+        throw new Error('Failed to fetch movie images');
+      }
+  
+      const data = await response.json();
+      movieImages.value = data; 
+     
+    } catch (error) {
+      movieImages.value=error;
```
Comment (Vijayshree-ranium) [inline src/utils/fetchMovieImages.js:14]: Space

**Review by Vijayshree-ranium — APPROVED** [review]

File: src/Views/MovieDetails.vue (hunk 2 above)
Comment (Deep687) [reply src/Views/MovieDetails.vue:62]: Done.

**Pull request merged**